pymongo/common.py

```diff
@@ -450,7 +472,8 @@ def validate_ok_for_update(update):
     'tz_aware': validate_boolean_or_string,
     'uuidrepresentation': validate_uuid_representation,
     'connect': validate_boolean_or_string,
-    'minpoolsize': validate_non_negative_integer
+    'minpoolsize': validate_non_negative_integer,
+    'unicode_decode_error_handler': validate_unicode_decode_error_handler
```

Comment:
There's a mix of parameter styles. You can see that tz_aware is snake-cased. The principle, more or less, is that standard MongoDB URI options are camel-cased, and PyMongo-specific options are snake-cased. This isn't deliberate, but it's a consequence of how we use the same code to validate MongoClient() parameters and URI options, and we've decided to live with it.

Short answer: let's leave unicode_decode_error_handler snake-cased.